src/gui/ui/imgui_gui.cpp

```diff
@@ -99,50 +102,23 @@ imgui_entry(World& world) {
         // your application based on those two flags.
         glfwPollEvents();
 
-        // TODO
-        // https://stackoverflow.com/questions/23362497/how-can-i-resize-existing-texture-attachments-at-my-framebuffer
+        if (!io.WantCaptureKeyboard) {
+            controls::computeMatricesFromInputs(window);
+        }
 
         glfwGetWindowSize(window, &window_width, &window_height);
         main_scene.update(window_width, window_height);
 
-        glm::vec3 position = controls::get_position_vector();
+        position = controls::get_position_vector();
 
-        FrameBufferHandler::instance().bind_fbo(0);
+        // "render" scene to the screen
+        main_scene.copy_to_window(window_width, window_height);
```

Comment:
how does this work now with IMGUI?

Reply:
This is how IMGUI is made to work. The widgets from IMGUI are placed over whatever is already there.